Mathlib/NumberTheory/Multiplicity.lean

```diff
@@ -96,9 +96,7 @@ theorem odd_sq_dvd_geom_sum₂_sub (hp : Odd p) :
             (∑ x ∈ Finset.range p, a ^ (x - 1) * (a ^ (p - 1 - x) * (↑p * (b * ↑x)))) +
           mk (span {s}) (∑ x ∈ Finset.range p, a ^ (x + (p - 1 - x))) := by
       ring_nf
-      simp only [← pow_add, map_add, Finset.sum_add_distrib, ← map_sum]
-      congr
-      simp [pow_add a, mul_assoc]
+      simp_rw [← map_sum, sum_add_distrib, map_add]
```

Comment:
I am utterly confused. This is not what I see when I open the file on this line?? Is this a bug in the github UI and one just needs to merge master again? Or am I missing something...

I see

```lean
simp only [add_mul, map_add, Finset.sum_add_distrib, ← map_sum]
congr <;> ext <;> ring
```

on that line in the code, what's happening here?

Reply:
Ah, just a problem on not having the latest commit locally, my bad.